gulpfile.js

```diff
@@ -391,30 +413,22 @@ function release_win(arch) {
 
 // Create distribution package (zip) for windows and linux platforms
 function release(arch) {
-    var src = path.join(appsDir, pkg.name, arch);
-    var output = fs.createWriteStream(path.join(releaseDir, get_release_filename(arch, 'zip')));
-    var archive = archiver('zip', {
-        zlib: { level: 9 }
-    });
-    archive.on('warning', function (err) { throw err; });
-    archive.on('error', function (err) { throw err; });
-    archive.pipe(output);
-    archive.directory(src, 'Betaflight Configurator');
-    return archive.finalize();
+    var src = path.join(appsDir, pkg.name, arch, '**');
+    var output = get_release_filename(arch, 'zip');
+
+    return gulp.src(src)
```

Comment:
In my test I've observed that in this way, we add the folder to the zip. So we must specify a base parameter, for example:
> return gulp.src(src, { base: path.join(appsDir, pkg.name, arch) })

If we want to create a `Betaflight Configurator` folder inside the zip (until now this is done in this way) ,we can do it calling a `rename` function in the pipe.

Reply:
The change to do this was a little more complex than expected. I needed to change the `gulp-archiver` by a `gulp-zip`. The first was not using the `pipe` correctly, so the rename throws errors of `file not found`. 
The https://github.com/betaflight/betaflight-configurator/pull/833 thas is merged makes all the changes needed, so when you `rebase` the code (and fight again with conflicts) this will be solved.